apollo-router/src/executable.rs

```diff
@@ -33,6 +33,16 @@ use crate::router::SchemaSource;
 use crate::router::ShutdownSource;
 use crate::EntitlementSource;
 
+// Note: We want to use jemalloc on unix, but we don't enable it if dhat-heap is in use because we
+// can only have one global allocator
+#[cfg(target = "unix")]
+use tikv_jemallocator::Jemalloc;
```

Comment:
NIT: this should be written in main.rs rather than executable.rs. If the router is used as a library, the library should not decide which allocator is used in the end

Reply:
done: 64c1ca5c2ec3db520a250f46424cf23ffd3d99d4